uk_results/migrations/0030_populate_postresult_post_election.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from collections import defaultdict
+
+from django.db import migrations, models
+
+
+def set_post_election_from_post(apps, schema_editor):
+    """
+    This is far from idea. Try to guess the PostExtraElection
+    that this PostResult relates to. This will have to be done by looking
+    and the related memberships and assuming they're correct (sometimes they
+    won't be, and that will have to be fixed manually later).
+    """
+    PostResult = apps.get_model('uk_results', 'PostResult')
+    PostExtraElection = apps.get_model('candidates', 'PostExtraElection')
+
+    print(PostResult.objects.all().count())
+    qs = PostResult.objects.all().select_related('post__extra')
+    for post_result in qs:
+        pee = None
+        elections = post_result.post.extra.elections.all()
+        if elections.count() == 1:
```

Comment:
Although there are no cases in the database where a post has no elections, I'd still stick in a `count() == 0` and raise an exception.